buildkite/bazelci.py

```diff
@@ -624,6 +624,13 @@
 
 SKIP_TASKS_ENV_VAR = "CI_SKIP_TASKS"
 
+# TODO: change to USE_BAZEL_DIFF once the feature has been tested in QA
+USE_BAZEL_DIFF_ENV_VAR = "EXPERIMENTAL_USE_BAZEL_DIFF"
+
+AUTO_DIFFBASE_VALUES = frozenset(["1", "true", "auto"])
```

Comment:
Do different values have different meanings, can you add a comment to explain it a bit?

Reply:
No, they are synonyms for each other

Reply:
Got it!